Underanalyzer/VMData.cs

```diff
@@ -608,11 +630,13 @@ public enum VariableType : byte
     /// <summary>
     /// The extended opcode of this instruction, if <see cref="Kind"/> is <see cref="Opcode.Extended"/>.
     /// </summary>
+    // TODO: should extended opcode have some default value akin to nil?
```

Comment:
It's probably more work than it's worth to define nil values for all of these (especially for enums where it's hard to do so, and it would potentially use more memory) - for now, they're just undefined behavior.

Reply:
For me i was thinking about having a clearer way of showing what is actually currently used, without needing to go through everything

Reply:
Yeah we can probably do a pass on the decompiler to see which instructions use which values, then document on each field which instructions would use them, and when.

Reply:
then i'll remove them for now.